pkg/osquery/table/platform_tables_windows.go

```diff
@@ -16,6 +16,8 @@ import (
 
 func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, currentOsquerydBinaryPath string) []osquery.OsqueryPlugin {
 	return []osquery.OsqueryPlugin{
+		dataflattentable.TablePluginExec(client, logger,
```

Comment:
need to add
`"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"`
to to the imports section above